### PR TITLE
[CMS-880] Remove proxysql compatibility issue notes

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

The proxysql fix was merged Monday, Aug 22. See [BUGS card](https://getpantheon.atlassian.net/browse/BUGS-5158?focusedCommentId=122625) for more info.

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7633](https://pr-7633--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `46` | `45` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7633` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41096/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41096/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7633--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/partials/_php-versions.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Click the links below to display complete PHP information for each version, incl
 | Version                                          | Available   | Recommended |
 | ------------------------------------------------ | :---------: | :---------: |
 | [8.1](https://v81-php-info.pantheonsite.io/)\*   | <span style="color:green">✔</span>         | -           |
```
Comment (scottbuscemi) [inline source/partials/_php-versions.md:7]: @jspellman814 Why don't we recommend 8.1? I thought that was also pending proxysql.

Comment (jspellman814) [reply source/partials/_php-versions.md]: It is, good catch! Updated in [fe00910](https://github.com/pantheon-systems/documentation/pull/7633/commits/fe00910bf0210916c765e977e9d04991e2ff9217).

**Comment by jspellman814** [comment]
[Preview](https://pr-7633--pantheon-docs.my.pantheonfrontend.website/docs/supported-php)

**Review by kporras07 — APPROVED** [review]
Looks good

**Comment by whitneymeredith** [comment]
LGTM

**Pull request merged**